### PR TITLE
Added more scenarios for hostnames abrt plugin

### PR DESCRIPTION
**Pull request by tkolhar**
please review it
thanks

### CODE REVIEW

**Comment by cswiii** [comment]
Additional tests, per demo:

* Ability to edit URL
* Ability to handle gracefully situations where URL isn't responding 
* Ability to handle gracefully situations where auth (username/passwd) is incorrect
* Assertion that managed systems (somehow?) get the proper auth stuff set up in abrt.conf  (both registered and newly-provisioned).  
* Possibly others...

**Comment by tkolhar** [comment]
requesting you to please review it
thanks

File: tests/foreman/cli/test_abrt.py [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+from ddt import ddt
+from fauxfactory import gen_string
+from robottelo.common.decorators import data
 
 
+def positive_create_data_1():
+    """Random data for positive creation"""
+
+    return (
+        {'hostname': gen_string("latin1", 10)},
```
Comment (elyezer) [inline tests/foreman/cli/test_abrt.py]: 10 is the default length, you can safely use `gen_string('str_type')`

**Comment by elyezer** [comment]
ACK, pending @cswiii's suggestions.

**Comment by cswiii** [comment]
Please assure we have test cases for the above comment implemented. Might as well get these all in. I don't want to ACK/merge and then lose the recommendations above.

File: tests/foreman/cli/test_abrt.py [hunk 2]
```diff
@@ -58,7 +74,29 @@ def test_abrt_3(self):
...
+        1. UI => Settings => Abrt tab => edit hostnames
+
+        @Assert: Assertion of hostnames is possible
+
+        @Status: Manual
+
+        """
+
+    @data({'hostname': ''},
+          {'hostname': 'space {0}'.format(gen_string('alpha', 10))},
```
Comment (elyezer) [inline tests/foreman/cli/test_abrt.py]: missed removing length of 10 here

File: tests/foreman/cli/test_abrt.py [hunk 3]
```diff
@@ -58,7 +74,29 @@ def test_abrt_3(self):
...
+          {'hostname': gen_string('alpha', 101)},
+          {'hostname': gen_string('html', 10)})
```
Comment (elyezer) [inline tests/foreman/cli/test_abrt.py]: missed removing length of 10 here

**Comment by tkolhar** [comment]
closing this as abrt is not shipped 
please reopen this in case of any queries

**Pull request closed without merging**